### DISCUSSION TITLE
Welcome to agar.io-clone Discussions!

### DISCUSSION THREAD

**Discussion by owenashurst** [huytd/agar.io-clone#533] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re thinking about.
  * Share ideas and new features.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by masonvick** [comment]
I just stumbled on this repo because I was interested in it, but I'm having trouble using the code. I can't seem to install the dependencies or get it to run. Any help would be great, thanks!

**Reply by owenashurst** [reply]
Which versions of Node/NPM are you running? Type `node -v` and `npm -v` into the command line.

**Comment by NobreMateus** [comment]
Hello, congratulations for the project. I tested and it is amazing!!

**Reply by owenashurst** [reply]
Thank you! Glad it's all working ok. Contributions are welcome, there's a lot of work to refactor the code at the minute.

**Comment by AlinaEisert** [comment]
Hello there, I am new to coding, so I wanted to do an Agar.io clone. Not exactly of cause, since I just wanted to learn some basics about game dev and nothing like js, css or web client stuff, which is why I still have no clue of how that works or how js or css are structured. Since I wanted to have similar aesthetics as the original Agar.io tho (e.g. squishing of cells, when they touch each other or the rim, or the wiggly borders of every element) i wanted to look up the code and maybe try to understand how it was done. I cannot seem to find the according file tho. So I wanted to ask: Is this done in this clone too? Or is the correct file to search in the "styles"-file in the original Agar.io code (F12 in Chrome)? And if so, how does it work? How could I recreate sth similar with C# or Python?
